active/Expander/Expander.md

```diff
@@ -0,0 +1,166 @@
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+## Notable Properties
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ExpandDirection | Sets the direction of expansion | Down = 0 |
+| IsExpanded | Whether or not control is expanded | False |
+
+## Events
+| Name | Description | 
+| :---------- | :------- | 
+| Expanded | Occurs when expanded |
+| Collapsed| Occurs when collapsed |
+
+# API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+~~~~
+Public enum ExpandDirection 
+{ 
+Down = 0 
+Up = 1 
+} 
+ 
+public class Expander : ContentControl 
+{ 
+Expander(); 
+ 
+public object Header {get;set;} 
+public DataTemplate HeaderTemplate { get;set; } 
+public DataTemplate HeaderTemplateSelector {get;set;} 
+ 
+public static readonly DependencyProperty HeaderProperty; 
+public static readonly DependencyProperty HeaderTemplateProperty; 
+public static readonly DependencyProperty HeaderTemplateSelectorProperty {get;} 
+ 
+public bool IsExpanded { get;set} 
+public ExpandDirection ExpandDirection { get;set;} 
+ 
+protected virtual void OnExpanded(); 
+protected virtual void OnCollapsed(); 
+ 
+public event Expanded; 
+public event Collapsed; 
+ 
+public static readonly DependencyProperty IsExpandedProperty; 
+public static readonly DependencyProperty ExpandDirectionProperty; 
+} 
+~~~~
+
+## Theme Resources
+
+| Name | Description | 
+| :---------- | :------- | 
+| ExpanderHeaderBackground | Header background color| 
+| ExpanderChevronBackground | Chevron background color| 
+| ExpanderChevronForeground | Chevron foreground color| 
+| ExpanderChevronMargin | Chevron margin thickness| 
+| ExpanderChevronGlyph | Chevron glyph| 
+| ExpanderChevronWidth | Chevron width (should this be renamed to 'weight'?)| 
+| ExpanderPopinVerticalOffset | vertical offset | 
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+# Inputs and Accessibility
+## UI Automation Patterns
+Expander will use a ExpandCollapsePattern. Expanding/Collapsing the expander will raise [RaisePropertyChanged](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.peers.automationpeer.raisepropertychangedevent?view=winrt-19041) with the property changed being the [ExpandCollapseProperty](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.expandcollapsepatternidentifiers.expandcollapsestateproperty?view=winrt-19041) property.
+
+## Keyboard
+* Only the header takes focus (not the entire Expander, or the chevron)
```

Comment:
What about scenarios when the content includes controls like the Button or Hyperlink?

Reply:
Since the header takes focus and not the chevron, after the entire header is focused, content inside the header takes focus (you can see this behavior in the Expander today 😄 https://github.com/microsoft/microsoft-ui-xaml/tree/master/dev/Expander )